ingest/anndata_.py

```diff
@@ -35,3 +44,65 @@ def validate(self):
         except ValueError:
             return False
 
+    @staticmethod
+    def generate_cluster_header(adata, clustering_name):
+        """
+        Based on clustering dimensions, write clustering NAME line to file
+        """
+        dim2 = ['NAME', 'X', 'Y']
+        dim3 = ['NAME', 'X', 'Y', 'Z']
```

Comment:
Slightly less duplicative:

```suggestion
        dim3 = dim2 + ['Z']
```

Reply:
`9bc7f61`